File: Trabajo#2/C++/Trabajo#12.cpp
#include <iostream>
using namespace std;

int main (){
	int edad,altura,peso;
	
		cout<<"Ingrese su edad porfavor: "<<endl;
	cin>>edad;
		cout<<"Ingrese su estatura porfavor: "<<endl;
	cin>>altura;
		cout<<"Ingrese su peso porfavor: "<<endl;
	cin>>peso;
	
	if(edad<=18){
		cout<<"Cumples con la edad requerida"<<endl;
	}
	
	else {
		cout<< "no cumples con la edad requerida"<<endl;
	}
	
		if(altura>=180){
		cout<<"Cumples con la estatura requerida"<<endl;
	}
	
	else {
		cout<< "no cumples con la estatura requerida"<<endl;
	}
	
		if(peso<=80){
		cout<<"Cumples con el peso requerido"<<endl;
	}
	
	else {
		cout<< "no cumples con el peso requerido"<<endl;
	}
	
	if(edad<=18){
	   (altura>=180);
	   (peso<=80);
	   cout<<"Bienvenido al club"<<endl;
	}
	else{
		cout<<"Lo sentimos intentlo a la proxima"<<endl;
	}
	
     
		
}

File: Trabajo#2/Python/Trabajo#12.py
n1 = int(input("ingrese la edad: "))
n2 = int (input("Ingrese la estatura: "))
n3 = int(input("ingrese el peso: "))

if n1<=18:
    print("Cumple con la edad")
else:
    print("No cumple con la edad")
if n2>180:
    print("Cumple con la estatura")
else:
    print("No cumple con la estatura")
if n3<=80:
    print("Cumple con el peso")
else:
    print("No cumples con el peso")
if (n1<=18 and n2>180 and n3<80):
    print("bienvenido al club")
else:
    print ("Lo sentimos intentalo a la proxima")
